README.md

```diff
@@ -222,11 +233,11 @@ the fact that we hit the same Endpoint definition three times. Let's refactor by
 finding the kind of `Impl Endpoint_1 IO`
 
 ```haskell
-type Impl_Endpoint =
-       IO (Response '[] 'Empty)
-  :<|> IO (Response '['H.Location '::: RawText] 'Empty)
-  :<|> IO (Response '[] ('Body '[Ct.TextPlain] Text))
-  :<|> IO NotHere
+type Impl_Endpoint_ = 
```

Comment:
What should I do here? I don't need Impl_Endpoint_, but I'm not sure what the preceding paragraph should be changed to, as there is no longer a need to refactor. Can I simply get rid of everything from ~194 to ~246?